OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/FirebaseAnalytics.java

```diff
@@ -471,4 +471,36 @@ public void trackWhatsNewSeen(@NonNull String versionName, int totalScreens) {
         event.putInt(Keys.TOTAL_SCREENS, totalScreens);
         logFirebaseEvent(event.getName(), event.getBundle());
     }
+
+    @Override
+    public void trackSubsectionVideosDelete(@NonNull String courseId, @NonNull String subsectionId) {
+        final FirebaseEvent event = new FirebaseEvent(Events.VIDEOS_SUBSECTION_DELETE, Values.VIDEOS_SUBSECTION_DELETE);
+        event.putString(Keys.COURSE_ID, courseId);
+        event.putString(Keys.SUBSECTION_ID, subsectionId);
+        logFirebaseEvent(event.getName(), event.getBundle());
+    }
+
+    @Override
+    public void trackSubsectionUndoVideosDelete(@NonNull String courseId, @NonNull String subsectionId) {
+        final FirebaseEvent event = new FirebaseEvent(Events.VIDEOS_UNDO_SUBSECTION_DELETE, Values.VIDEOS_UNDO_SUBSECTION_DELETE);
+        event.putString(Keys.COURSE_ID, courseId);
+        event.putString(Keys.SUBSECTION_ID, subsectionId);
+        logFirebaseEvent(event.getName(), event.getBundle());
+    }
+
+    @Override
+    public void trackUnitVideoDelete(@NonNull String courseId, @NonNull String unitId) {
+        final FirebaseEvent event = new FirebaseEvent(Events.VIDEOS_UNDO_UNIT_DELETE, Values.VIDEOS_UNDO_UNIT_DELETE);
```

Comment:
Wrong event name and bi value is getting passed here.
It should be **VIDEOS_UNIT_DELETE**

Reply:
Good catch!